bedrock/mozorg/models.py

```diff
@@ -169,3 +176,23 @@ class AdvertisingIndexPage(AbstractBedrockCMSPage):
     ]
 
     template = "mozorg/cms/advertising/advertising_index_page.html"
+
+
+class TwoColumnSubpage(AbstractBedrockCMSPage):
```

Comment:
Nit, nonblicking: if this is advertising-only, should the class name reflect that?